matrix/plugin.video.umbrella/resources/lib/modules/player.py

```diff
@@ -748,7 +748,9 @@ def get(self, name, imdb, season, episode):
 			content = server.DownloadSubtitles(token, content)
 			content = base64.b64decode(content['data'][0]['data'])
 			content = gzip.GzipFile(fileobj=BytesIO(content)).read()
-			subtitle = control.transPath('special://temp/')
+			download_path = jsloads(control.jsonrpc('{"jsonrpc":"2.0", "method":"Settings.GetSettingValue", "params":{"setting":"subtitles.custompath"}, "id":1}')).get('result').get('value')
+			download_path = 'special://subtitles/' if download_path is not '' else 'special://temp/'
+			subtitle = control.transPath('download_path')
```

Comment:
should be
`			subtitle = control.transPath(download_path)`

otherwise you will get 

[2023-01-19 14:54:21] [ plugin.video.umbrella: DEBUG ]: downloaded subtitle=Puss.in.Boots.The.Last.Wish.2022.1080p.WEBRip.x265-RARBG.srt
[2023-01-19 14:54:21] [ plugin.video.umbrella: DEBUG ]: subtitle file = download_path\TemporarySubs.he.srt

and the file isn't saved